numpy/core/include/numpy/npy_cpu.h

```diff
@@ -39,6 +39,7 @@
      * _M_AMD64 defined by MS compiler
      */
     #define NPY_CPU_AMD64
+    #define NPY_CPU_X86_64
```

Comment:
Why do we need both defines?

Reply:
Just an alias to make everyone happy, I used the term "X86_64" in all infrastructure work for x86 64bit mode but it's okay I can remove it and use only `NPY_CPU_AMD64` since the current definition `NPY_CPU_AMD64` already been used in another header `numpy/core/include/numpy/npy_endian.h`  .